src/comm/comm.h

```diff
@@ -235,6 +235,26 @@ typedef struct {
   ExtParameter param;
 } LidarExtParameter;
 
+typedef struct {
+  float roll;  /**< Roll angle, unit: degree. */
+  float pitch; /**< Pitch angle, unit: degree. */
+  float yaw;   /**< Yaw angle, unit: degree. */
+} FilterTransformParameter;
+
+typedef struct {
+  std::string filter_frame_id;  /**< Frame */
+  float filter_yaw_start;  /**< Start yaw angle, unit: degree. */
+  float filter_yaw_end;  /**< End yaw angle, unit: degree. */
```

Comment:
maybe it makes sense to name it filter_sensor_yaw_start and _end? because everything which is not inside start and end will be filtered out, so it defines more the sensor yaw values.

Reply:
Hm, I think from an "outsider" user point of view it would be easier to define the yaw filter range based off the yaw axis from `base_link`, i.e. if I say filter Yaw range is 90 - 180° then this is the range that is filtered, meaning the lower left quarter is filtered out. This would make sense for a sensor that is mounted in front right corner for example.